src/client/src/rt-util/getAppName.ts

```diff
@@ -1,10 +1,31 @@
 import { getEnvironment } from 'rt-util'
+import { currencyFormatter } from './'
+import { appTitles } from '../index'
```

Comment:
I am not sure we should import the app titles here? Can we just move them here?

Reply:
Yeah I can move it all here @rikoe